main.go

```diff
@@ -382,6 +388,17 @@ func main() {
 		return c.String(http.StatusOK, string(data))
 	})
 
+	e.GET("/gohan/data-types", gohanRequestPublic)
+
+	e.GET("/katsu/data-types", func(c echo.Context) error {
```

Comment:
this also needs to filter out counts, no?

Reply:
I thought was only for gohan. But sure!

Reply:
counts are in both responses to my knowledge